_data/languages.yml

```diff
@@ -1084,6 +1427,24 @@
     morphosyntax: [ isolating ]
   territories: [ ]
 
+-
+  codes: [ yua ]
+  names: [ Yucatec Maya ]
+  family: [ myn ]
+  scripts: [ Latn ]
+  typology:
+    word_order: [ VOS ]
+  territories: [ mx ]
+-
+  codes: [ yue ]
+  names: [ Cantonese, Yue, Yue Chinese ]
+  family: [ sit ]
+  scripts: [ Hans, Hant ]
+  typology:
+    word_order: [ SVO ]
+    morphosyntax: [ analytic, isolating ]
+  territories: [ hk, mo ]
```

Comment:
`cn` too

Reply:
This is a tricky one, because the code is technically for a group of languages, one of which happens to be more famous.